vector2tile_restart_mod.f90

```diff
@@ -739,6 +743,10 @@ subroutine WriteTileRestart(namelist, date, tile)
       (/dim_id_xdim,dim_id_ydim,dim_id_time/), varid)
       if (status /= nf90_noerr) call handle_err(status)
 
+    status = nf90_def_var(ncid, "snodl", NF90_DOUBLE,   &
```

Comment:
Won't this crash if either snodl or snwph is not in the file? 

I think we need to us snd_name here.

Reply:
I see yes - just definition, thanks. 

That's a good point. It's included as both in FV3 model output, with snwdph being the non-land mask specific version (so a diagnostic, really). No harm including both here - but perhaps we can clean this up later when we remove the <GFSv17 code.